arch/arm/boot/dts/qcom/qcom-msm8974-sony-xperia-rhine-togari.dts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qcom-msm8974-sony-xperia-rhine.dtsi"
+
+/delete-node/ &touchscreen;
+
+/ {
+	model = "Sony Xperia Z Ultra";
+	compatible = "sony,xperia-togari", "qcom,msm8974";
+	chassis-type = "handset";
+
+	gpio-keys {
+		key-camera-snapshot {
+			status = "disabled";
+		};
+
+		key-camera-focus {
+			status = "disabled";
+		};
+	};
+};
+
+&blsp2_i2c2 {
+	status = "disabled";
+
+	clock-frequency = <355000>;
+	vdd-supply = <&pm8941_l22>;
+
+	touchscreen: maxim_max1187x_tsc@48 {
```

Comment:
Make it `touchscreen@48`. Also FYI when you send this upstream, you need to remove this section since the max1187x driver is not upstream